docs/source/alert_development_guide.rst

```diff
@@ -141,15 +137,66 @@ and add your new foo alert to the others with a crontab style schedule
     'unauth_ssh.AlertUnauthSSH': {'schedule': crontab(minute='*/1')},
   }
 
-Restart your MozDef instance and you should begin seeing alerts on the alerts page.
+The format is 'pythonfilename.classname': {'schedule': crontab(timeunit='*/x')} and you can use any celery time unit (minute, hour) along with any schedule that makes sense for your environment.
+Alerts don't take many resources to execute, typically finishing in sub second times, so it's easiest to start by running them every minute.
+
+How to run the alert in the docker containers
+----------------------------------------------
+Once you've got your alert passing tests, you'd probably like to send in events in a docker environment to further refine, test, etc.
+
+
+There are two ways to go about integration testing this with docker:
+1) Use 'make run' to rebuild the containers each time you iterate on an alert
+2) Use docker-compose with overlays to instantiate a docker environment with a live container you can use to iterate your alert
+
+In general, the 'make run' approach is simpler, but can take 5-10mins each iteration to rebuild the containers (even if cached).
+
+To use the 'make run' approach, you edit your alert. Add it to the docker/compose/mozdef_alerts/files/config.py file for scheduling as discussed above and simply:
+::
+  make run
+
+This will rebuild any container that needs it, use cache for any that haven't changed and restart mozdef with your alert.
+
+
+
+To use a live, iterative environment via docker-compose:
+::
+
+  docker-compose -f docker/compose/docker-compose.yml -f docker/compose/dev-alerts.yml -p mozdef up
+
+This will start up all the containers for a mozdef environment and in addition will allow you an adhoc alerts container to work in that loads the /alerts directory as a volume in the container.
+To run the alert you are developing you will need to edit the alerts/files/config.py file as detailed above to schedule your alert. You will also need to edit it to reference the container environment as follows
```

Comment:
The correct file to modify is actually 'alerts/lib/config.py', not 'alerts/files/config.py'.

Reply:
good catch! hence the guide ;-]